homework/index.js

```diff
@@ -30,18 +30,100 @@
     return elem;
   }
 
+  function createInfoDiv(repoDiv, selectRepo) {
+    const tableDiv = createAndAppend('div', repoDiv, { class: 'left-div' });
+    const infoTable = createAndAppend('table', tableDiv, { class: 'contributor-table' });
+    const tableBody = createAndAppend('tbody', infoTable);
+    const rowName = createAndAppend('tr', tableBody);
+    const repoName = createAndAppend('td', rowName, { text: 'Repository:', class: 'repo-name' });
+    createAndAppend('a', repoName, {
+      target: '_blank',
+      href: selectRepo.html_url,
+      text: selectRepo.name,
+    });
+    const rowDesc = createAndAppend('tr', tableBody);
+    createAndAppend('td', rowDesc, {
+      text: `Description: ${selectRepo.description}`,
+      class: 'repo-desc',
+    });
+    const rowFork = createAndAppend('tr', tableBody);
+    createAndAppend('td', rowFork, {
+      text: `Forks: ${selectRepo.forks}`,
+      class: 'repo-Fork',
+    });
+    const rowUpDate = createAndAppend('tr', tableBody);
+    createAndAppend('td', rowUpDate, {
+      text: `updated: ${selectRepo.updated_at}`,
+      class: 'repo-update',
+    });
+  }
+  function createContributorDiv(root, selectedRepo, repoDiv) {
+    const contributorsUrl = selectedRepo.contributors_url;
+    fetchJSON(contributorsUrl, (err, contributors) => {
+      if (err) {
+        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+        return;
+      }
+      const contributorsDiv = createAndAppend('div', repoDiv, { class: 'right-div' });
+      createAndAppend('p', contributorsDiv, { text: 'Contributors', class: 'contributors-header' });
+      contributors.forEach(contributor => {
+        const contributorsList = createAndAppend('ul', contributorsDiv, { class: 'list' });
```

Comment:
You should create only one `ul` for the complete list of contributors. Then, for each contributor add a `li` to that single `ul`.